doc/connectivity/networking/api/wifi.rst

```diff
@@ -30,37 +30,63 @@ Wi-Fi PSA crypto supported build
-in PEM format are committed to the repo at :zephyr_file:`samples/net/wifi/test_certs` and the during the
+Wi-Fi enterprise security requires use of X.509 certificates, two methods of installing certificates are supported:
+
+Compile time certificates
+-------------------------
+
+Test certificates in PEM format are committed to the repo at :zephyr_file:`samples/net/wifi/test_certs` and the during the
 build process the certificates are converted to a C header file that is included by the Wi-Fi shell
 module.
 
+If you want to use your own certificates, you can replace the existing certificates with your own certificates in the same directory.
+
+.. code-block:: bash
+
+    $ export WIFI_TEST_CERTS_DIR=samples/net/wifi/test_certs/rsa3k
+    $ cp client.pem $WIFI_TEST_CERTS_DIR
+    $ cp client-key.pem $WIFI_TEST_CERTS_DIR
+    $ cp ca.pem $WIFI_TEST_CERTS_DIR
+    $ cp client2.pem $WIFI_TEST_CERTS_DIR
+    $ cp client-key2.pem $WIFI_TEST_CERTS_DIR
+    $ cp ca2.pem $WIFI_TEST_CERTS_DIR
+    $ west build -p -b <board> samples/net/wifi -S wifi-enterprise
+
+or alternatively copy ``rsa2k`` certificates by changing the ``WIFI_TEST_CERTS_DIR`` environment variable.
+
 .. code-block:: bash
 
-    $ cp client.pem samples/net/wifi/test_certs/
-    $ cp client-key.pem samples/net/wifi/test_certs/
-    $ cp ca.pem samples/net/wifi/test_certs/
-    $ cp client2.pem samples/net/wifi/test_certs/
-    $ cp client-key2.pem samples/net/wifi/test_certs/
-    $ cp ca2.pem samples/net/wifi/test_certs/
-    $ west build -p -b <board> samples/net/wifi -- -DEXTRA_CONF_FILE=overlay-enterprise.conf
+    $ export WIFI_TEST_CERTS_DIR=samples/net/wifi/test_certs/rsa2k
+
+or you can set the :envvar:`WIFI_TEST_CERTS_DIR` environment variable to point to the directory containing your certificates.
+
+.. code-block:: bash
+
+    $ west build -p -b <board> samples/net/wifi -S wifi-enterprise -- -DWIFI_TEST_CERTS_DIR=<path_to_your_certificates>
+
+Run time certificates
+---------------------
+
+The Wi-Fi shell module uses TLS credentials subsystem to store and manage the certificates. The certificates can be added at runtime using the shell commands, see :ref:`tls_credentials_shell` for more details.
+The sample or application need to enable the :kconfig:option:`CONFIG_WIFI_SHELL_RUNTIME_CERTIFICATES` option to use this feature.
 
-For using variable size network buffer, the following overlay file can be used:
+To facilitate installation of the certificates, a helper script is provided in the ``samples/net/wifi/test_certs`` directory. The script can be used to install the certificates at runtime.
 
 .. code-block:: bash
 
-    $ west build -p -b <board> samples/net/wifi -- -DEXTRA_CONF_FILE=overlay-enterprise-variable-bufs.conf
+    $ samples/net/wifi/test_certs/install_certs.py -p samples/net/wifi/test_certs/rsa2k
 
+The script will install the certificates in the ``rsa2k`` directory to the TLS credentials store in the device over UART and using TLS credentials shell commands.
```

Comment:
so all devices are supporting the AT commands this tool seems to be using?

Reply:
This doesn't use AT command, but TLS credentials shell (part of Zephyr)